### DISCUSSION TITLE
LCD Landscape

### DISCUSSION THREAD

**Discussion by Frohmut95** [mriscoc/Ender3V2S1#102] [Q&A] [answered]
Hey, is there an Option to rotate the Standard Ender 3 V2 Display to landscape mode? in the Marlin Files where some snippets of code for this but couldnt find a way.

**Comment by mriscoc** [comment, marked as answer]
For Landscape you must use the official MarlinUI Landscape mode.

**Reply by Frohmut95** [reply]
I builded the one from otioss a while back, so can I implement the tweaks he mentioned in your Marlin code and it would work? Sorry for asking im relativly new to mixing firmware fetarures :D

**Reply by Frohmut95** [reply]
So when i add MARLINUI Landscape and define it in the configuration.h everthing gets broken. tThe Rotate function with proui doesnt do anythng either

**Reply by mriscoc** [reply]
ProUI is only available with the DWIN_LCD_PROUI, and there isn't a landscape mode for ProUI, for landscape you have the option DWIN_MARLINUI_LANDSCAPE which uses the Marlin standard menus and parameters, you must use the sources from the official Marlin repository.

**Reply by SamueldaCostaAraujoNunes** [reply]
Hello! I hope you are doing well, I would like to implement the UI in landscape mode. Could you give me some help on where I should start looking?

**Reply by mriscoc** [reply]
Check MarlinUI Landscape mode, but not all the display units support the screen rotation for landscape.
